tmux.conf

```diff
@@ -56,12 +56,12 @@ bind-key [ copy-mode
 bind-key ] paste-buffer
 
 # Setup 'v' to begin selection as in Vim
-bind-key -t vi-copy v begin-selection
-bind-key -t vi-copy y copy-pipe "reattach-to-user-namespace pbcopy"
+bind-key -T copy-mode-vi v send-keys -X begin-selection
+bind-key -T copy-mode-vi y send-keys -X copy-pipe "reattach-to-user-namespace pbcopy"
```

Comment:
This will more similarly replicate the previous behavior from before 2.4 if you change `copy-pipe` to `copy-pipe-and-cancel`, so that pressing `y` will copy to the system buffer then exit copy mode and bring you back down to input mode in the terminal